text/3455-t-test.md

```diff
@@ -0,0 +1,110 @@
+- What parts of [cargo nextest](https://nexte.st/) can we stabilize?
+- With the proliferation of test frameworks (e.g. [rstest], [trybuild], [trycmd], [cargo_test_support], [criterion]), are there underlying needs we can resolve?
+
+## Relationships to other teams
+
+With the aforementioned breadth across the Project, the Testing team will need to have collaborative relationships with many other teams, and is conceptually a subteam of both the Libs and Dev Tools teams.
+
+The rust-lang/team repo does not currently support representing dual-parent subteams, so for now the Testing team will be primarily under the Dev Tools team
+
+T-devtools: This will be the primary top level parent team.
+
+**T-cargo**: This is a sibling team that T-testing will need to work with similarly to T-rustfmt, T-clippy, etc.
+
+**T-rustdoc**: This is a sibling team that T-testing will likely coordinate with if any changes are need to how we do doctesting
+
+**T-IDEs and Editors**: This is a sibling team that T-testing will likely coordinate with to understand the needs of IDEs/editors related to incorporating test related capabilities
+
+**T-libs**: This will be a second/secondary top level parent team as they are ultimately responsible for libtest.
+
+## Processes
+
+For decisions on vision and direction, T-testing will use a standard FCP process.  T-testing will be subject to [T-cargo's processes](https://doc.crates.io/contrib/team.html#decision-process) when dealing with `cargo test` and T-libs's processes for libtest.  For any newly developed crates and libraries, we will follow [T-cargo's processes](https://doc.crates.io/contrib/team.html#decision-process).
+
+## Membership
+
+Team members are expected to shepherd testing discussions and vote on FCPs. Team membership is independent of regular code contributions though contributing can help build up the relevant experience and/or demonstrate the qualities for team membership.
+
+Qualifications that will be taken into account for membership include:
+
+- Does the person have the judgement for when deciding when a larger consensus is needed?
+- Does the person understand the constraints of backwards compatibility within `cargo test` and libtest and exercise caution when extending the compatibly constraints?
+
+Someone can become a member of the Testing Team by requesting a review or being nominated by one of the existing members. They can be added by unanimous consent of the team. The team lead or another member of the team will also confirm with the moderation team that there are no concerns involving the proposed team member.
+
+Team Leads are responsible for scheduling and facilitating team meetings and will be selected from the team members by consensus.
+
+The initial members of the Testing team shall be:
+- Lead: Caleb Cartwright (@calebcartwright)
+- Ed Page (@epage)
+- Weihang Lo (@weihanglo)
+- Scott Schafer (@Muscraft)
```

Comment:
Just wonder what the process is to have people here if somebody wants to join as an initial member?

Reply:
I suspect it might be easiest if we start with a smaller group to get through the RFC process and then the team can use the documented process to add more members.  I know of at least one person I want to reach out to once the team is up and running.

That said, I'd be fully on-board with a T-libs member signing up to be an initial member so we have closer contacts with that team.

Reply:
> I'd be fully on-board with a T-libs member signing up to be an initial member so we have closer contacts with that team.

I'm willing to join and serve as the liaison with t-libs. I have a lot of interest here, as I've been disappointed in the current state of libtest, but unsure how to move it forward.

Reply:
Yeah having an explicit liaison relationship would be good and was something I was hoping to see!

Reply:
Ah, I'm not libs-api, and while initially it said t-libs, it now says t-libs-api. So I'll withdraw my offer in case someone on that team would rather server that role.

Reply:
t-libs-api was my own thought on this but if others think t-libs is also need or would be be better, I'm fine with it

Reply:
I think both/eithre libs or libs-api is fine. I would very much encourage having Thom on the team.

Reply:
Same!

Reply:
thomcc is now in the list!